vita3k/gui/src/gui.cpp

```diff
@@ -711,6 +711,8 @@ void pre_init(GuiState &gui, EmuEnvState &emuenv) {
     init_font(gui, emuenv);
     lang::init_lang(gui.lang, emuenv);
 
+    auto &io = ImGui::GetIO();
+
```

Comment:
Is there a reason for this (or if GetIO initialises some stuff, there is no reason to assign its output to a variable)

Reply:
This is my mistake. That part remained when I was removing unnecessary code after adding different code to that section and confirming it was not needed.